pkg/cli/cmds/stage.go

```diff
@@ -6,5 +6,5 @@ const (
 	// coredns and servicelb run controllers that are turned off when their manifests are disabled.
 	// The k3s CloudController also has a bundled manifest and can be disabled via the
 	// --disable-cloud-controller flag or --disable=ccm, but the latter method is not documented.
-	DisableItems = "coredns, servicelb, traefik, local-storage, metrics-server"
+	DisableItems = "coredns, servicelb, traefik, traefik-crd, local-storage, metrics-server"
```

Comment:
This seems a bit too granular to me. What does it mean to:
- `--disable=traefik` but not also `--disable=traefik-crd`?
- `--disable=traefik-crd` but also not `--disable=traefik`?

It would seem to me that if you want traefik you will want it to work and hence the CRD is a requirement that should be silently installed.